i18n/shared/src/main/scala/io/udash/i18n/types.scala

```diff
@@ -1,24 +1,6 @@
 package io.udash.i18n
 
-import com.avsystem.commons.serialization.GenCodec
-
 case class Lang(lang: String) extends AnyVal
```

Comment:
Very minor, but you may want to take a look at `@transparent` annotation for `GenCodec`